atqa-framework/tests/pages/AccountPage.ts

```diff
@@ -0,0 +1,20 @@
+import { BasePage } from './BasePage';
+
+export class AccountPage extends BasePage {
```

Comment:
As far as I know, the idea of page object is to provide an interface for interacting with elements, e.g:
```typescript
// in test
accountPage = AccountPage(I)
accountPage.clickGetStartedButton()
accountPage.setBothPasswords('some_password_value")
// etc..

```
But this complicates some aspects of development (you need to support api of page object)

I think in this case we can simplify the interface and store selectors as class attributes, for example:
```typescript
class AccountPage extends BasePage{
    getStartedBtn = locate('button').withText('Get Started')
    passwordCreateInput = { xpath: '(//input)[1]' }
    passwordConfirmInput = { id: 'second' }
    continueBtn = locate('button').withText('Continue')
    // etc..
}

Reply:
If we made some structural changes to a complex component/page, you need to update the nested structure for the page object, this can be tricky.

My idea is to have a page object class that contains:
* unambiguous element names
* a locator to find it on that page

Reply:
the nested structure needs to provide more clear situation around exact pages cause locator which has been named just **locatorPasswordButton** - means nothing and hard to future maintaining and it also touches for list

but locator object 

`AUTH_FORM: {
      PASSWORD_INPUT: '#input_1',
      CONFIRM_PASSWORD_INPUT: '#input_2'
}
`
means a lot, a lot more than just **passwordInput**, it also provide an ability to group locators objects within one page 

the pages contains a lot of logic and locators should be grouped and structured thus for more easier interact with each page.

upper case makes such objects unique and provide a 100% understanding what is exactly does such object which could be declared in any place of framework.

it doesn't make developing harder at all but on the contrary, it allows you to simplify page navigation and code maintaining